config.env.py

```diff
@@ -31,5 +31,12 @@
 LDAP_BIND_DN = environ.get("PACKET_LDAP_BIND_DN", None)
 LDAP_BIND_PASS = environ.get("PACKET_LDAP_BIND_PASS", None)
 
+# Mail Config
+MAIL_PROD = environ.get("PACKET_MAIL_PROD", False)
+MAIL_SERVER = environ.get("PACKET_MAIL_SERVER", "thoth.csh.rit.edu")
+MAIL_USERNAME = environ.get("PACKET_MAIL_USERNAME", "packet@csh.rit.edu")
+MAIL_PASSWORD = environ.get("PACKET_MAIL_PASSWORD", None)
+MAIL_USE_TLS = environ.get("PACKET_MAIL_PASSWORD", True)
```

Comment:
```suggestion
MAIL_USE_TLS = environ.get("PACKET_MAIL_TLS", True)
```

Reply:
Wait where does this even get used? Does flask mail just magically pull it from the app config? Hmmmm.

Reply:
Yeah a lot of flask extension automatically pull from `app.config` so as long as you define the variables it works